### ISSUE TITLE
Vircadia-builder interface error

### ISSUE THREAD

**Issue by ugrozkr** [vircadia/vircadia#1443]
I try to build source code with Vircadia-builder. All processes finished successfully, then I clicked Vircadia Desktop icon it do not do anything, I checked runtime file on run_interface file and I received following error. 
![Görüntü 11 11 2021 01 14](https://user-images.githubusercontent.com/21170360/141202969-17b8d08f-549a-4699-a5fb-b14cdf4cdf5e.jpg)

**Comment by ugrozkr** [comment]
hi, Can anyone help my problem ?

**Comment by ksuprynowicz** [comment]
It looks like a virtual machine. Does it have accelerated OpenGL ES 3.2?

**Comment by ugrozkr** [comment]
Yes i use VM with Ubuntu 18.04 LTS image. It working on VMware Fusion and OpenGL 3.3, Mesa 20.0.8 are installed on this VM. 
![Görüntü 13 11 2021 17 55](https://user-images.githubusercontent.com/21170360/141648410-90114811-3504-4e97-bc2c-982127f19709.jpg)

**Comment by ksuprynowicz** [comment]
Does the official AppImage build run on it?

**Comment by daleglass** [comment]
Hello! I fixed a very similar looking problem for Ubuntu 20.04 recently. I'll try porting my fixes back to 18.04.

However, the interface requires OpenGL 4.1. It probably still won't work after fixing that.

**Comment by ugrozkr** [comment]
@daleglass thanks for you support, Vircadia working well on Ubuntu 20.04, this problem caused for only Ubuntu 18.04 LTS, @ksuprynowicz you should update Vircadia-builder document, 18.04 is not supported fully.

Virtual Machine is not recommend, resources are limited and that is also get many memory leaks at VM side.
